src/dmd/backend/cod1.d

```diff
@@ -3078,6 +3078,11 @@ void cdfunc(ref CodeBuilder cdb, elem* e, regm_t* pretregs)
         if (alignsize > stackalign &&
             (I64 || (alignsize == 16 && tyvector(ep.Ety))))
         {
+            if (tyvector(ep.Ety) && alignsize > STACKALIGN)
```

Comment:
I don't think the tyvector() clause is necessary.

Reply:
I64 includes user defined types.

Reply:
Yes, but if their alignsize is larger than STACKALIGN, shouldn't they get so aligned, too? After all, what if the vector type is a field in a struct?

Reply:
or an array of vectors?

Reply:
True.